interop/src/main.rs

```diff
@@ -59,7 +59,7 @@ fn run_test() -> Result<()> {
     use tokio::net::{TcpListener, TcpStream};
 
     color_eyre::install()?;
-    if std::env::var("RUST_LOG").is_ok() || std::env::var("CI").is_ok() {
+    if std::env::var("RUST_LOG").is_ok() {
```

Comment:
We don't really care if the contents of this variable are unicode or not, but this still parses them. Using `var_os.is_some()` is a microoptimization, but technically it's better.